documentation/src/docs/asciidoc/release-notes/release-notes-5.1.0-M2.adoc

```diff
@@ -54,6 +54,13 @@ on GitHub.
 ** For example, an `@Inherited` annotation such as `@TestInstance` will now be discovered
    on a custom _composed annotation_ declared on a superclass even if the _composed
    annotation_ is not declared as `@Inherited`.
+* New Kotlin friendly `fail` methods added as _top-level functions_ in the
+  `org.junit.jupiter.api` package.
+ - When calling the original methods from Kotlin the compiler required the generic return
```

Comment:
We don't use a dash for nested bullet points but rather `**`.

Reply:
Replaced with `**` in https://github.com/junit-team/junit5/pull/1213/commits/6ea0807ddb79a2f508f849862732ee34d1e498e6